core/tests/test_utils.py

```diff
@@ -1443,6 +1446,8 @@ def swap_with_call_counter(
         if call_through:
             impl = obj.attr
         else:
+            # Here we use type Any because this method returns the return value
+            # of swapped function and that value can be of any type.
```

Comment:
```suggestion
            # Here we use type Any because this method returns the return value
            # of the swapped function, and that value can be of any type.
```

A lot of these comments have grammar issues. Please proofread and fix the comments you're changing, especially the new ones you're adding

Reply:
tried my best to fix these nits, PTAL!